curriculum/challenges/_meta/lab-stack-class/meta.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Build a Stack Class",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "blockType": "lab",
+  "dashedName": "lab-stack-class",
+  "order": 264,
```

```suggestion
  "order": 263,
```

Looks like the order changed for this one.

---

Updated :+1: